maat-court-data-api/src/main/java/gov/uk/courtdata/entity/UserRoleEntity.java

```diff
@@ -0,0 +1,46 @@
+package gov.uk.courtdata.entity;
+
+import gov.uk.courtdata.model.id.UserRoleId;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@IdClass(UserRoleId.class)
+@Table(name = "USER_ROLES", schema = "TOGDATA")
+public class UserRoleEntity {
+    @Id
```

Comment:
Do you want to double-check the @Id annotation on username and role name?

Reply:
Discussed and agreed that this is not an issue, since this approach is already being used in other entity classes and works well.